src/content/mods.cpp

```diff
@@ -109,6 +128,10 @@ void parseModContents(ModSpec &spec)
 			std::vector<std::string> dependencies;
 
 			std::ifstream is((spec.path + DIR_DELIM + "depends.txt").c_str());
+
+			if (is.good())
+				log_mod_deprecation(spec, "depends.txt is deprecated, please use mod.conf instead.");
```

Comment:
If `mod.conf` exists and has a `depends`/`optional_depends` key, and `depends.txt` also exists, no warning will be produced.  Same goes for `description.txt`.  Is this intended?

Reply:
Oh, never mind, I see.  This is to allow mods to work on older versions of Minetest where the txt files were necessary without deprecation notices on newer versions.